src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -432,15 +434,29 @@ private AdsDataProvider() {
   }
 
   @DataProvider
-  public static Object[][] kruxStandardSegment() {
+  public static Object[][] kruxSegments() {
     return new Object[][]{
         {
-            "glee",
-            "Rachel",
-            "adtest",
-            "SyntheticTests/Krux/Page_1",
-            "pqdapsy7l"
-        }
+            "J-RIfJI0",
+            new Page("vim", "Vim_Tips_Wiki"),
+            "[\"pqdapsy7l\"]", // Standard segment for visiting adtest before
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ATrailingCommentCheck)